src/components/ShareButton.tsx

```diff
@@ -5,12 +5,19 @@ import HKSMButton from "./HKSMButton";
 interface Props {
   id: string;
   onClick: () => void;
+  text: string;
   disabled?: boolean;
 }
 
-const ShareButton: React.FC<Props> = ({ id, onClick, disabled }: Props) => {
+const ShareButton: React.FC<Props> = ({
+  text,
```

Comment:
`ShareButton` should be responsible for its own text. Feel free to keep the change, but I'd prefer "Share" be hardcoded in this component